README.md

```diff
@@ -13,18 +18,60 @@ A pure Go implementation of the [HDR Histogram](https://github.com/HdrHistogram/
 
 For documentation, check [godoc](http://godoc.org/github.com/codahale/hdrhistogram).
```

Comment:
this requires change to point to new doc url

Reply:
Your totally right @ahothan, fixed on the new commit :)